e2eshark/README.md

```diff
@@ -164,8 +164,9 @@ Note that the --cachedir command line argument is necessary for any run command.
 Run the tests in operators, combinations folders of the default framework (i.e. pytorch),
 Use framework to onnx to torch MLIR path (--mode onnx) and run upto inference (default) using llvm-cpu backend (default),
 use four processor cores (default --jobs 4) on your machine, generate report file after finishing test run
+If the model you are running requires a huggingface token (llama, gemma), set the HF_TOKEN env variable as well.
```

Comment:
User could set environment in shell as export HF_TOKEN=<token> or provide it on command line launching run.py, maybe mention both?